src/app/service-box/service-box.component.html

```diff
@@ -0,0 +1,66 @@
+<button mat-icon-button (click)="opened = !opened">
+	<mat-icon>apps</mat-icon>
+</button>
+
+
+<div class="box" [class.opened]="opened">
+	<div class="tip">
+		<div class="border"></div>
+		<div class="foreground"></div>
+	</div>
+
+	<div class="services">
+		<div class="row">
+			<a href="https://api.loklak.org/">
+				<div class="col">
+					<div class="img">
+						<img src="assets/images/cow_39x45.png" width="45" height="52" alt="Loklak Server">
+					</div>
+					<span>Loklak</span>
+				</div>
+			</a>
```

Comment:
This might be a bit confusing. We are showing loklak logo, writing loklak but the link is pointing to loklak API page. What can be a better way to do this?

Reply:
Being at the Loklak search sit itself, I'd suggest we don't mention it only. This services box is for the other services offered and I feel doesn't require it to mention Loklak again

Reply:
Please Suggest what to write here

Reply:
We can just write Loklap API? @singhpratyush

Reply:
@anshumanv That'd be a good idea but the component is for services offered yeah? Not APIs right? I'd say mention the API in the footer and not as a service cause it's just for developers

Reply:
Makes sense, it would be suited better in the footer @ajayns :+1:

Reply:
Thanks @ajayns for suggestions , I am removing this link from here and will link it in footer in next PR. :+1

Reply:
Hey @ParthS007 I think there is already a link to the API in the footer, please check and update. :smile:

Reply:
Ohh  updated the comment 👍